streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -771,11 +771,14 @@ private long adjustRecordsProcessedBeforeCommit(final long prevRecordsProcessedB
     protected void maybeCommit(final long now) {
 
         if (commitTimeMs >= 0 && lastCommitMs + commitTimeMs < now) {
-
-            log.info("{} Committing all active tasks {} and standby tasks {} because the commit interval {}ms has elapsed by {}ms",
-                logPrefix, activeTasks.keySet(), standbyTasks.keySet(), commitTimeMs, now - lastCommitMs);
+            log.debug("{} Committing all active tasks {} and standby tasks {} since {}ms has elapsed (commit interval is {}ms)",
+                    logPrefix, activeTasks.keySet(), standbyTasks.keySet(), now - lastCommitMs, commitTimeMs);
 
             commitAll();
+
+            log.info("{} Committed all active tasks {} and standby tasks {} in {}ms",
```

Comment:
I think this is better stayed as `INFO`, and actually we are even more `chatty` that we have one entry per task, whereas now we are using one line indicating how much time has been elapsed for committing all tasks. If we want to make it DEBUG then I agree both this and the above should be debugging as you suggested, but my personal preference is keep it `INFO` since such events are quite important.